PittsburghServices/pittsburgh_services_scraper.py

```diff
@@ -80,8 +80,8 @@ def grab_data(self):
     service_summary="",
     check_collection="services",
     dump_collection="tmpPittsburghServices",
-    dupe_collection="tmpPittsburghServicesFoundDuplicates",
-    data_source_collection_name="pittsburgh_services",
+    dupe_collection="tmpPittsburghServicesDuplicates",
+    data_source_collection_name="PittsburghServicesScraper",
```

Comment:
lets use snake case here. Also remove word scraper from because real source is pittsburgh services , scraper is the way to obtain that data. :)